packages/framework-integration-tests/integration/fixtures/events/post-created.ts

```diff
@@ -3,11 +3,7 @@ import { UUID } from '@boostercloud/framework-types'
 
 @Event
 export class PostCreated {
-  public constructor(
-    readonly postId: UUID,
-    readonly title: string,
-    readonly body: string,
-  ) {}
+  public constructor(readonly postId: UUID, readonly title: string, readonly body: string) {}
```

Comment:
I'm not sure if you change the fixtures, the integration test pass. Have you run integration tests after this formatting?